cleverhans/attacks.py

```diff
@@ -116,6 +183,11 @@ def __init__(self, model, back='tf', sess=None):
         Create a FastGradientMethod instance.
         """
         super(FastGradientMethod, self).__init__(model, back, sess)
```

Comment:
Maybe we don't. I'm only seeing iteration over `feedable` now.
It doesn't hurt anything to make it an `OrderedDict`, and then we're protected if someone else starts iterating over it in the future, but I won't insist on it.